### DISCUSSION TITLE
Repo access

### DISCUSSION THREAD

**Discussion by sreichel** [OpenMage/magento-lts#3683] [General]
From my packagist accunt ... 

:upside_down_face: 

> [openmage/legacy-frontend-themes](https://packagist.org/packages/openmage/legacy-frontend-themes) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/dev-copyright](https://packagist.org/packages/openmage/dev-copyright) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/dev-translations](https://packagist.org/packages/openmage/dev-translations) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/dev-meta-package](https://packagist.org/packages/openmage/dev-meta-package) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/module-mage-pagecache](https://packagist.org/packages/openmage/module-mage-pagecache) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/module-mage-backup](https://packagist.org/packages/openmage/module-mage-backup) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))
> 
> [openmage/magento-lts](https://packagist.org/packages/openmage/magento-lts) GitHub user has no admin access to the repository, or Packagist was not granted access to the organization ([check here](https://github.com/settings/connections/applications/a059f127e1c09c04aa5a))

I dont even have access to projects ive created?

Well done ....

**Comment by sreichel** [comment]
@Flyingmana, you beg for money ... ? For a project you did nearly nothin for?

**Reply by Flyingmana** [reply]
thats an unrelated for the topic personal attack, repeating will have a temporary block on organization level as a consequence for violating the Code of Conduct.

This will as a technical side effect also remove al permissions on organizational repositories, including the ones you created in the name of the organization.

**Reply by sreichel** [reply]
@Flyingmana please tell me what you did over years for this project.

You created a bank account for funding. All money goes to you right now, Without any permission. For a project you did nothing for.

> will have a temporary block 

This really scares me after be being normal member ...  not.


Daniel, wenn Geld auf das Konto eingeht ....

**Reply by Flyingmana** [reply]
> All money goes to you right

I need to correct this, its not going to me, or a bank account which is owned by me. The Concept of OpenCollective financial hosts is, that this is managed by an external entity, which then expenses are submitted and they settle it with the collected Money.

> @Flyingmana please tell me what you did over years for this project.

definitely a valid request to ask what I did, I get these from time to time, so to have it easier in the future (so I only need to link it then) I wrote it this time in a more structured way here: https://flyingmana.wordpress.com/2023/12/03/openmage-2023-a-personal-recap/

Iam sadly a bit slower, so it took me 3 evenings to finish this summary.
When we speak about money which actually goes to me, according to the current draft of the spending plan https://github.com/OpenMage/organizational/blob/main/finance/expense_planning.md
I see myself only in the second level for the "maintaining the organisatorical and transparency information".
In numbers this would mean, I could expect 500$ per month, when we reached a founding goal of 100,000$ per Year. But not if by then someone else took over this responsibilities, or the community voted for a different Plan.
In comparison and in business terms, this would mean around 6% of the funding go into Operations.

But I do thank you for critically questioning the plans, this helps to make them more robust and offers me a chance to proof how solid the current concept is.

**Comment by m-overlund** [comment]
All this publicly sarcasm, internal conflicts etc. has to stop.

**Reply by sreichel** [reply]
That guy that did nearly nothing to that project is calling for money now ...

Sarcasm?

**Reply by m-overlund** [reply]
> Sarcasm?

This is sarcastic:

> Well done ....

---

> That guy that did nearly nothing to that project is calling for money now ...

I'm not about who's in the right or wrong.
This sarcasm, bitterness and conflict does not help anyone or the project, which is why we're here.

There are better ways to express frustration or disagree with decisions.
